ui/pages/confirmations/utils/date.ts

```diff
@@ -0,0 +1,28 @@
+const months = [
+  'January',
+  'February',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'August',
+  'September',
+  'October',
+  'November',
+  'December',
+];
+
+export const formatDate = (dateString: string) => {
+  if (!dateString) {
+    return dateString;
+  }
+  const date = new Date(dateString);
```

Comment:
Could we achieve this easier with `Intl.DateTimeFormat.format` since, plus it will translatable?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/format

There seems to be some existing usages in `ui/helpers/utils/notification.util.ts`.

Reply:
I tried that, but I was facing issues in displayed same value of day in different locales.

Reply:
My worry is what we have currently is partially re-implementing that, plus we have no translation or locale support at all.

Can you share the issue concerning locales?

Reply:
Please check this broken test case: https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/84899/workflows/000a7574-6f5e-4e06-bc7b-fd50448a2c3d/jobs/3066779

Reply:
I've tested with that commit, and I believe the issue was just that we weren't explicit with the timezone.

Using the following in format date, the existing util and snapshot tests passed without any changes to them:

```
export const formatDate = (dateString: string) => {
  if (!dateString) {
    return dateString;
  }

  return DateTime.fromISO(dateString)
    .setLocale('en')
    .setZone('utc')
    .toFormat('dd LLLL yyyy, HH:mm');
};
```

We can of course choose to use the user's timezone and or their locale, but as a first step, this means we don't rely on any custom format logic and have total control of the output.

Reply:
Good suggestion, I updated the PR.